docs/metrics/micrometer-connector.md

```diff
@@ -0,0 +1,93 @@
+---
+id: micrometer-connector
+title: Micrometer Connector
+---
+
+ZIO ZMX also has an integration with [Micrometer](https://micrometer.io/), a powerful metrics instrumentation library.
```

Comment:
ZIO ZMX is a separated and deprecated project https://github.com/zio/zio-zmx

Looks like this project is based on that one

Reply:
sure! you are right, ZMX mention should be totally removed from docs